x-pack/platform/plugins/shared/event_log/server/event_logger.ts

```diff
@@ -161,6 +186,17 @@ function logEventDoc(logger: Logger, doc: Doc): void {
   logger.info(`event logged: ${JSON.stringify(doc.body)}`);
 }
 
+function logUpdateEventDoc(logger: Logger, docs: Array<Required<Doc>>): void {
+  logger.info(`event updated: ${JSON.stringify(docs.length)}`);
```

Comment:
this should be a `debug` ...

Reply:
Sorry, I missed the context.  This is actually used when the "logging" config is set for the event log, where it will dump the doc's it writes to the Kibana log.  So it probably makes sense for it to be an `info`.  However, I don't believe anyone uses this in practice.  It was added a debugging tool for EL developers, so might be useful to have as info, but not a big deal.